diff --git a/bgfx/tools/shaderc/shaderc_metal.cpp b/bgfx/tools/shaderc/shaderc_metal.cpp
index e8fd208..7f7a9a2 100644
--- a/bgfx/tools/shaderc/shaderc_metal.cpp
+++ b/bgfx/tools/shaderc/shaderc_metal.cpp
@@ -23,7 +23,6 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
 #include <glslang/Public/ShaderLang.h>
 #include <glslang/Include/ResourceLimits.h>
 #include <glslang/SPIRV/GlslangToSpv.h>
-#include <glslang/SPIRV/SPVRemapper.h>
 #include <glslang/SPIRV/SpvTools.h>
 #include <spirv-tools/optimizer.hpp>
 BX_PRAGMA_DIAGNOSTIC_POP()
 
diff --git a/bgfx/tools/shaderc/shaderc_spirv.cpp b/bgfx/tools/shaderc/shaderc_spirv.cpp
index a844cc0..8b6de8b 100644
--- a/bgfx/tools/shaderc/shaderc_spirv.cpp
+++ b/bgfx/tools/shaderc/shaderc_spirv.cpp
@@ -22,7 +22,6 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
 #define ENABLE_OPT 1
 #include <glslang/Public/ShaderLang.h>
 #include <glslang/Include/ResourceLimits.h>
-#include <glslang/SPIRV/SPVRemapper.h>
 #include <glslang/SPIRV/GlslangToSpv.h>
 #include <glslang/SPIRV/SpvTools.h>
 #include <spirv-tools/optimizer.hpp>
 BX_PRAGMA_DIAGNOSTIC_POP()
