diff --git a/cmake/bgfx/shaderc.cmake b/cmake/bgfx/shaderc.cmake
index 1f4ed2b..a1f49e2 100644
--- a/cmake/bgfx/shaderc.cmake
+++ b/cmake/bgfx/shaderc.cmake
@@ -25,7 +25,7 @@ target_link_libraries(
 	PRIVATE bx
 			bgfx-vertexlayout
 			fcpp
 			bimg
-			glslang::glslang glslang::glslang-default-resource-limits glslang::SPIRV glslang::SPVRemapper
+			glslang::glslang glslang::glslang-default-resource-limits glslang::SPIRV
 			glsl-optimizer
 			spirv-cross-core spirv-cross-reflect spirv-cross-glsl spirv-cross-hlsl spirv-cross-msl
 			webgpu
 )
